packages/markdown/__tests__/index.test.js

```diff
@@ -175,3 +176,94 @@ test('should strip dangerous img attributes', () => {
     '<img src="x"/>',
   );
 });
+
+describe('export multiple Markdown renderers', () => {
+  const { dash, hub, ast, md, html } = markdown.render;
+  const text = `# Hello World`;
+  const tree = {
```

Comment:
https://jestjs.io/docs/en/snapshot-testing is your friend here!

Reply:
wow, well this turned out to be an _extremely_ useful suggestion @domharrington!